dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/DefaultTrackerImportService.java

```diff
@@ -82,12 +79,10 @@ private PersistenceReport commit(TrackerImportParams params, TrackerBundle track
   @IndirectTransactional
   public ImportReport importTracker(
       TrackerImportParams params, TrackerObjects trackerObjects, JobProgress jobProgress) {
-    User user = trackerUserService.getCurrentUser();
```

Comment:
I think we should assert that there is a current user at the start of our import. It would be a programmer error not to have it set. We should rather fail at the start of the import than later down the line.

Reply:
Generally I would agree, but I think this case is an exception.
Since we are not validating user input, I think we should assume it's always present and use it as such.